pkg/kudoctl/cmd/package_list_plans_test.go

```diff
@@ -0,0 +1,60 @@
+package cmd
+
+import (
+	"bytes"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPackageListPlans(t *testing.T) {
+	file := "plans-list"
+	out := &bytes.Buffer{}
+	cmd := newPackageListPlansCmd(fs, out)
+	if err := cmd.RunE(cmd, []string{"../packages/testdata/zk.tgz"}); err != nil {
+		t.Fatal(err)
+	}
+
+	gp := filepath.Join("testdata", file+".golden")
+
+	if *updateGolden {
```

Comment:
Probably not yours, but why would we update files in a test? Especially if it's the file we're testing against?

Reply:
it is to easy the burden on "known" output changes... it's a pretty standard technique